android/BOINC/app/src/main/java/edu/berkeley/boinc/client/Monitor.kt

```diff
@@ -796,6 +919,54 @@ class Monitor : LifecycleService() {
         }
     }
 
+        private val mRpcExternBroadCastReceiver = RpcExternBroadcastReceiver()
```

Comment:
An anonymous class can be used instead (use `object : BroadcastReceiver()`).

Reply:
Maybe, but this is this seems to be the preferred way